cmd/publish.go

```diff
@@ -0,0 +1,65 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cmd
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/elastic/elastic-package/internal/cobraext"
+	"github.com/elastic/elastic-package/internal/github"
+	"github.com/elastic/elastic-package/internal/publish"
+)
+
+const publishLongDescription = `Use this command to publish a new package revision to the Package Storage.
+
+The command checks if the package hasn't been already published to the package-storage (whether it's present in snapshot/staging/production branch or open as pull request). If the package revision hasn't been published, it will open a new pull request.
```

Comment:
Let's try to not talk about `package-storage` here if possible. We should start treating it more like an implementation detail (persistent store for the package registry) and talk in terms of the package registry instead.

Reply:
Fixed